### ISSUE TITLE
zooming effect is not working in mobile screen

### ISSUE THREAD

**Issue by albinjoseph1994** [wittlock/ngx-image-zoom#46]
The zooming effect is not working on the mobile screen. In the computer screen its works perfectly

**Comment by wittlock** [comment]
Yeah, there's currently no support for touch screen events. Not sure how well I can get all the modes to work for touch screen but I think at least one of them should work. I'll look into adding some support.

**Comment by 4javier** [comment]
any news about this issue?

**Comment by pavankumarT345** [comment]
Does Zooming image works for mobile Screen also .If not  can anyone suggest other alternatives to zoom image in mobile view

**Comment by mikila85** [comment]
+1
